modules/e2ee/ManagedKeyHandler.js

```diff
@@ -41,10 +41,14 @@ export class ManagedKeyHandler extends KeyHandler {
             this._onParticipantPropertyChanged.bind(this));
         this.conference.on(
             JitsiConferenceEvents.USER_JOINED,
-            this._onParticipantJoined.bind(this));
+            async () => {
```

Comment:
Do you need to do this? Making `_onParticipantJoined` async and using await inside should be enough no?